client/my-sites/sharing/connections/mailchimp-settings.jsx

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { requestSettingsUpdate } from 'state/mailchimp/settings/actions';
+import QueryMailchimpLists from 'components/data/query-mailchimp-lists';
+import QueryMailchimpSettings from 'components/data/query-mailchimp-settings';
+import FormRadio from 'components/forms/form-radio';
+import FormLabel from 'components/forms/form-label';
+import FormFieldset from 'components/forms/form-fieldset';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const MailchimpSettings = ( {
+	siteId,
+	keyringConnections,
+	requestSettingsUpdateAction,
+	mailchimpLists,
+	mailchimpListId,
+} ) => {
+	const chooseMailchimpList = listId => {
+		requestSettingsUpdateAction( siteId, {
+			follower_list_id: listId,
+			keyring_id: keyringConnections[ 0 ].ID,
+		} );
+	};
+
+	return (
+		<div>
```

Comment:
Do you need this `<div>` for the UI? If you're creating it just to have a container for the `Query` components, you can squeeze them into the `FormFieldset` equally well.